pkg/serverless/otlp/otlp.go

```diff
@@ -0,0 +1,96 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2021-present Datadog, Inc.
+
+//go:build otlp
+// +build otlp
+
+package otlp
```

Comment:
I don't think this warrants its own package. It's a very small one-off thing used exclusively in one place. Furthermore, it creates clashing with another existing package (in name). However, I don't feel strongly about this and will leave it completely up to you to decide what to do here.